cmd/outflux/migrate_i_test.go

```diff
@@ -42,18 +43,21 @@ func TestMigrateSingleValue(t *testing.T) {
 	rows, err := dbConn.Query("SELECT * FROM " + measure)
 	if err != nil {
 		t.Error(err)
+		return
 	}
 	defer rows.Close()
 
 	var time time.Time
 	var field1 int
 	if !rows.Next() {
 		t.Fatal("couldn't check state of TS DB")
+		return
```

Comment:
Is this necessary? I thought Fatal basically ends program execution?

Reply:
It's not, double checked